news_items/017_workspaces/info.json

```
@@ -0,0 +1,6 @@
+{
+    "uuid": "33f5fec9-e21c-422f-8ad2-b0d309bd72e6",
+    "title": "New PrairieLearn Workspaces",
```

```suggestion
    "title": "New PrairieLearn workspaces",
```
Looks like other news titles are sentence case. Or do we want to go ahead and brand "PL Workspaces" as a proper noun?